src/ads_impl.cc

```diff
@@ -390,10 +390,11 @@ void AdsImpl::ClassifyPage(const std::string& url, const std::string& html) {
   auto winner_over_time_category = GetWinnerOverTimeCategory();
 
   LOG(INFO) << "Site visited " << url << ", immediateWinner is "
-      << winning_category << " and winnerOverTime is "
-      << winner_over_time_category;
+            << winning_category << " and winnerOverTime is "
```

Comment:
Need to be indented by 4 characters (as was) to adhere to linter/coding standards